projects/RabbitMQ.Client/RabbitMQ.Client.csproj

```diff
@@ -67,7 +63,7 @@
     <PackageReference Include="MinVer" Version="2.4.0" PrivateAssets="All" />
     <PackageReference Include="System.Memory" Version="4.5.4" />
     <PackageReference Include="System.Threading.Channels" Version="5.0.0" />
-    <PackageReference Include="Microsoft.CodeAnalysis.NetAnalyzers" Version="5.0.1" />
+    <PackageReference Include="Microsoft.CodeAnalysis.NetAnalyzers" Version="5.0.3" />
```

Comment:
Do we know what version of the .NET SDK we're using to build against in CI? If we can just update to a .NET 5 SDK, I'd prefer to do that over explicitly adding a package reference to the analyzer.

Then we just need to have `<EnableNETAnalyzers>true</EnableNETAnalyzers>` in the project file to enable the default set of analyzers to run against this project.

Reply:
RabbitMQ's Concourse uses

```
dotnet-sdk-3.1
dotnet-runtime-2.1
```

I could not get .NET 5 build 3.1 apps without surprising failures (I no longer remember the details). I'd be all for moving to use .NET 5 (but target .NET Core 3.1+ for now) but perhaps it should be a separate PR.